developers/ocean-node/compute-to-data-c2d.md

```diff
@@ -0,0 +1,178 @@
+# Compute-to-data (C2D)
```

Comment:
Please check this comment: https://github.com/oceanprotocol/docs/pull/1447#discussion_r1691299910

Reply:
I still see here that c2d readme is not linked in this page.